controller/proxy-injector/webhook.go

```diff
@@ -71,13 +74,22 @@ func Inject(api *k8s.API,
 		} else {
 			parent = &objs[0]
 		}
+		ownerKind = strings.ToLower(ownerRef.Kind)
 	}
+	proxyInjectionAdmissionRequests.With(admissionRequestLabels(ownerKind, request.Namespace, configLabels)).Inc()
 
-	if injectable, reason := report.Injectable(); !injectable {
+	if injectable, reasons := report.Injectable(); !injectable {
+		var readableReasons, metricReasons string
+		metricReasons = strings.Join(reasons, ", ")
```

Comment:
For machine-readable label values, you might wanna avoid the space after the comma.

Reply:
Updated. :)